src/main/java/uk/gov/hmcts/reform/ethos/ecm/consumer/config/CosDBConfiguration.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.reform.ethos.ecm.consumer.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceProperties;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.jdbc.DataSourceBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import javax.sql.DataSource;
+
+@Configuration
+public class CosDBConfiguration {
+    @Value("${etcos.spring.datasource.url}")
+    private String url;
+    @Value("${etcos.spring.datasource.username}")
+    private String username;
+    @Value("${etcos.spring.datasource.password}")
+    private String password;
+
+    @Bean(name = "etcosDataSource")
+    public DataSource etcosDataSource() {
```

Comment:
Does it need the FlywayDataSource tag ?

Reply:
Deffo not, we don't want flyway operating on a database that isnt owned by message-handler. the migrations are handled by et-cos